### PR TITLE
Fix doc: kamelet connect via direct endpoint

### PR DESCRIPTION
**Pull request by llowinge**
Fixes wrong swapped definition of source/sink connection to direct endpoint + the formatting.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by llowinge** [comment]
@nicolaferraro @astefanutti

File: docs/modules/ROOT/pages/kamelets/kamelets.adoc [hunk 1]
```diff
@@ -497,8 +497,9 @@ spec:
 ----
 
 Source and sink flows will connect to the outside route via the `direct:{{routeId}}` endpoint:
-- A source Kamelet must start **from** `direct:{{routeId}}`
-- A sink Kamelet must contain a call **to** `direct:{{routeId}}`
+
+- A source Kamelet must contain a call **to** `direct:{{routeId}}`
```
Comment (nicolaferraro) [inline docs/modules/ROOT/pages/kamelets/kamelets.adoc:501]:
```suggestion
- A source Kamelet must contain a call **to** `kamelet:sink`
```

File: docs/modules/ROOT/pages/kamelets/kamelets.adoc [hunk 2]
```diff
@@ -497,8 +497,9 @@ spec:
...
+- A sink Kamelet must start **from** `direct:{{routeId}}`
```
Comment (nicolaferraro) [inline docs/modules/ROOT/pages/kamelets/kamelets.adoc:502]:
```suggestion
- A sink Kamelet must start **from** `kamelet:source` 
```

**Review by nicolaferraro — CHANGES_REQUESTED** [review]
This changed in the runtime.

@lburgazzoli isn't it a bit confusing? :D

**Comment by lburgazzoli** [comment]
> This changed in the runtime.
> 
> @lburgazzoli isn't it a bit confusing? :D

what is confusing ?

**Comment by llowinge** [comment]
@nicolaferraro Probably whole kamelets.adoc should be updated. I will close this one.

**Pull request closed without merging**